events/blocks/events-form/events-form.js

```diff
@@ -136,6 +136,7 @@ function createButton({ type, label }, bp) {
         button.classList.remove('submitting');
         if (!respJson) return;
 
+        if (respJson.ok) eventFormSendAnalytics(bp,'Form Submit');
```

Comment:
nit: this and similar occurrences -- we usually have a space between params.

Reply:
oh, sorry. Just realizing that since this is my first time in this repo, I should have run npm install to get the linter running properly. Doing that now and I will make these updates.